src/NextGen/WelcomeBanner/resources/views/welcome-banner.php

```diff
@@ -167,9 +182,14 @@
                         border: solid 1px var(--givewp-blue-500);
                         background-color: var(--givewp-blue-25);
                     ">
-                        <img style="width: 100%;" src="<?php esc_attr_e(GIVE_NEXT_GEN_URL . '/src/NextGen/WelcomeBanner/resources/design-mode.min.png'); ?>" alt="">
-                        <h3><?php _e('Design Mode', 'give'); ?></h3>
-                        <div><?php _e('See exactly what your form looks like for potential donors using the "Design" tab of the builder. Changes are visible immediately.', 'give'); ?></div>
+                        <img style="width: 100%;" src="<?php
+                        esc_attr_e(GIVE_NEXT_GEN_URL . '/src/NextGen/WelcomeBanner/resources/design-mode.min.png'); ?>"
```

Comment:
```suggestion
                        esc_attr(GIVE_NEXT_GEN_URL . '/src/NextGen/WelcomeBanner/resources/design-mode.min.png'); ?>"
```

Since this is a static string, and not a translatable string, we should use `esc_attr`.

Reply:
I'l keep this in mind. Resolved: https://github.com/impress-org/givewp-next-gen/pull/176/commits/79a44168d109f79f3ffeae7999bc7e6463a4a827